client/app/scripts/superdesk-authoring/views/article-edit.html

```diff
@@ -40,13 +40,13 @@
 
 <div class="field">
     <label translate>Dateline</label>
-    <div sd-meta-locators class="dateline-info" ng-show="_editable && !proofread"
+    <div sd-meta-locators class="dateline-info" ng-show="_editable"
```

Comment:
It is meant for that, but i don't want to switch back to normal mode for correcting a mistake.